NYC_Summit18_Developer_Workshop/idct/README.md

```diff
@@ -0,0 +1,141 @@
+Inverse Discrete Cosine Transform
```

Comment:
This file has lots of info that needs to be updated.

Reply:
This file is a README from SDAccel example directory.  Developers will be reviewing the IDCT.md file in the workshop for the idct example.  Will remove this file.